locale/de_DE/LC_MESSAGES/OpenInExplorer.po

```diff
@@ -0,0 +1,38 @@
+# OpenInExplorer Darktable Plug-In.
+# Copyright (C) 2020 Volker Lenhardt
+# This file is distributed under the same license as the PACKAGE package.
+# Volker Lenhardt <volker.lenhardt@unitybox.de>, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenInExplorer\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-04-14 14:35+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Volker Lenhardt <volker.lenhardt@unitybox.de>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: de_DE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: OpenInExplorer.lua:59
+msgid "OpenInExplorer plug-in only supports Windows and Linux at this time"
+msgstr "OpenInExplorer-Plug-in kann nur unter macOS, Linux oder Windows laufen"
+
+#: OpenInExplorer.lua:75
+msgid "Please select an image"
+msgstr "Bitte mindestens 1 Bild auswählen"
+
+#: OpenInExplorer.lua:85
+msgid "Please select fewer images (max 15)"
+msgstr "Bitte nicht mehr als 15 Bilder auswählen"
+
+#: OpenInExplorer.lua:93
+msgid "Show in file explorer"
+msgstr "Im Dateimanager anzeigen"
+
+#: OpenInExplorer.lua:95
+msgid "Opens File Explorer at the selected image's location"
+msgstr "Öffnet den Dateimanager mit dem ausgewählten Dateinamen"
```

Comment:
Nicht so richtig schön, was hältst du von:
"Öffnet den Dateimanager an der Position des ausgewählten Bildes"
evtl. fällt dir ja noch was besseres ein.

Reply:
Gefällt mir, was Besseres finde ich auch. Ich war schon mit meiner Lösung irgendwie nicht zufrieden.

Die Korrekturen werden ich, wie gesagt, nach der endgültigen Fassung des Scripts vornehmen.

Da ich neu beim lua-Scripting bin, weiß ich gar nicht, wie denn die Übersetzungsdateien in DT importiert werden. Ich finde dort nämlich nur po-Dateien, auch hier in GitHub. Soll der Endnutzer, wenn er die Übersetzung nutzen will, erst einmal gettext, damit auch msgfmt installieren und anwenden? Kann man nicht direkt die nötigen mo-Dateien liefern?

Reply:
Warum wir keine .mo files (mehr) haben:
https://github.com/darktable-org/lua-scripts/pull/147

Reply:
Ah, interessant. Werde mal gen_18n_mo.lua ausprobieren.